### ISSUE TITLE
Issues running develop

### ISSUE THREAD

**Issue by jmwenda** [caktus/django-timepiece#824]
Hey,

I have some issues running this on a development environment with the css/compression not working. On viewing the source page, the css is filed viewable but the styling does not apply on the webpages. Any pointers on the css issues.

**Comment by FFX01** [comment]
Did you make sure to install all dependencies and run ```python manage.py collectstatic``` before running your server?

**Comment by jmwenda** [comment]
Yep, I did so and css still does not load the bootstrap css. Any clue?

**Comment by jmwenda** [comment]
Quick question, seems I kept getting the error about the key.
You have offline compression enabled but key "8090182da58eb15ffb1a99034f0afb94" is missing from offline manifest. You may need to run "python manage.py compress".

Sadly on looking at the manifest. The values were totally different. I replaced the manifest file with the above value and it worked. Isn't it supposed to be generated and updated every time one runs compress?
